guacamole/src/main/webapp/app/login/directives/login.js

```diff
@@ -188,17 +190,36 @@ angular.module('login').directive('guacLogin', [function guacLogin() {
 
                     // Clear all remaining fields that are not username fields
                     angular.forEach($scope.remainingFields, function clearEnteredValueIfPassword(field) {
-
+                        
                         // If field is not username field, delete it.
                         if (field.type !== Field.Type.USERNAME && field.name in $scope.enteredValues)
                             delete $scope.enteredValues[field.name];
-
+                        
                     });
+
                 }
 
             }));
 
         };
+        
+        /**
+         * Returns the field most relevant to the user given the current state
+         * of the login process. This will normally be the first empty field.
+         *
+         * @return {Field}
+         */
```

Comment:
Missing documentation for the return value.

Reply:
> Update: Added the two pieces of documentation requested above.

I see the new documentation for `$scope.relevantField`, but this return value still appears undocumented.